include/exec/__detail/__sender_facade.hpp

```diff
@@ -109,7 +109,7 @@ namespace exec {
       if constexpr (__lacks_transform_sender<_Kernel>) {
         return __mtype<_Sender>{};
       } else {
-        if constexpr (__valid<__tfx_sender_, _Kernel, _Sender, _Env>) {
+        if constexpr (__valid_<__tfx_sender_, _Kernel, _Sender, _Env>) {
```

Comment:
what does `__valid` mean on msvc?

Reply:
Ah. It's in `<sal.h>` as well: https://gist.github.com/caiorss/0678b5efb2aa2eff120b9e9284fdbb0c#file-sal-h-L2448-L2460

Reply:
Could you rename `__valid` instead to `__mvalid`?

Reply:
Done.